onboard-app-guilherme/src/screens/login/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Button, Text, TextInput, View } from "react-native";
+
+export function LoginScreen() {
```

Comment:
Acho que não dá problema se tem um único export (que deveria ser sempre o caso para os arquivos de página mesmo), mas uma boa prática é sempre colocar o export default para os arquivos do expo-router
```suggestion
export default function LoginScreen() {
```